test/e2e/specs/editor/blocks/post-title.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post Title block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'Can edit the post title', async ( { editor, page } ) => {
+		// Create a block with some text that will trigger a list creation.
+		await editor.insertBlock( { name: 'core/post-title' } );
+
+		// Add the post title
+		await page.locator( 'role=textbox[name="Add title"i]' ).focus();
+		await page.keyboard.type( 'Just tweaking the post title' );
+
+		// Save the post draft and reload.
+		await page.locator( '.editor-post-save-draft' ).click();
```

Comment:
Let's change this to role selectors.

```suggestion
		await page.click( 'role=button[name="Save draft"i]' );
```